recipe/meta.yaml

```diff
@@ -129,21 +128,21 @@ outputs:
       binary_relocation: False
```

Comment:
Probably ok to keep this. Though quick question, do we have any libraries still here to corrupt? Only see the static library below, which I think `patchelf` shouldn't touch

Reply:
Agreed this isn't doing anything now. Left it in since the relocation only seems to break things for us.